### PR TITLE
Add dataset terms

### PR DESCRIPTION
**Pull request by khelm**
Added dataset and SIO terms to nidm-experiment.owl 
Reorganized to better reflect PROV-DM heirarchy of entity/agent/activity

### CODE REVIEW

File: nidm/nidm-experiment/terms/nidm-experiment.owl [hunk 1]
```diff
@@ -164,304 +170,898 @@ dct:title rdf:type owl:DatatypeProperty .
...
+
+:BehaviorAndConditionOnsets rdf:type owl:Class ;
+                            
+                            rdfs:subClassOf prov:Entity .
+
+
+
+###  http://purl.org/nidash/nidm/experiment#Behavioral_File_Format
+
+:Behavioral_File_Format rdf:type owl:Class ;
```
Comment (nicholsn) [inline nidm/nidm-experiment/terms/nidm-experiment.owl]: For new terms, the syntax we've been using for classes is `BehavioralFileFormat` without underscores - we only use underscores when importing external terms that already use this convention

File: nidm/nidm-experiment/terms/nidm-experiment.owl [hunk 2]
```diff
@@ -10,7 +10,7 @@
 @prefix xsp: <http://www.owl-ontologies.com/2005/08/07/xsp.owl#> .
 @prefix daml: <http://www.daml.org/2001/03/daml+oil#> .
 @prefix meta: <http://purl.obolibrary.org/obo/iao/dev/ontology-metadata.owl#> .
-@prefix nidme: <http://purl.org/nidash/nidm/experiment#> .
+@prefix nidm: <http://purl.org/nidash/nidm/experiment#> .
```
Comment (nicholsn) [inline nidm/nidm-experiment/terms/nidm-experiment.owl]: i think we are moving towards a single namespace for all nidm terms with a trailing '/':

`@prefix nidm: <http://purl.org/nidash/nidm/> .`

Comment (cmaumet) [inline nidm/nidm-experiment/terms/nidm-experiment.owl]: I agree with @nicholsn (as discussed in #237). Though, as this is not yet implemented in the `nidm-experiment.owl` file, I think we could wait for this PR to be merged before taking care of harmonising the namespaces.

**Comment by nicholsn** [comment]
@khelm at this point is this PR dead? Do you still want it merged or should we just close and move forward in another PR?

**Comment by khelm** [comment]
This one is dead - we should just close it as there an more recent pull request.

**Pull request closed without merging**